### PR TITLE
Add `init` script to @hiro-ui/react-scripts

### PR DESCRIPTION
**Pull request by miraage**
- `init` script was added (compatible with CRA --typescript flag)
- also added `prettier` dep to `@hiro-ui/prettier-config` as we did with `eslint` & `@hiro-ui/eslint-config`

### CODE REVIEW

File: packages/react-scripts/scripts/init.js [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+    'chrome >= 64',
+    'ie 11',
+    'edge >= 15',
+  ];
+
+  appPkg.eslintConfig = {
+    extends: ['@hiro-ui'],
+  };
+
+  appPkg.prettier = '@hiro-ui';
```
Comment (Drache93) [inline packages/react-scripts/scripts/init.js]: Has this been tested? I didn't think this format was supported for prettier, just eslint.

File: yarn.lock [hunk 2]
```diff
@@ -13502,7 +13502,7 @@ prettier-linter-helpers@^1.0.0:
   dependencies:
     fast-diff "^1.1.2"
 
-prettier@^1.17.1:
+prettier@^1.17.1, prettier@^1.18.2:
```
Comment (Drache93) [inline yarn.lock]: We can remove the prettier dep from the root package if it's in prettier config. Same with the eslint deps. Should work with just `lerna` as a dep of the root.

File: packages/react-scripts/scripts/init.js [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+  let args;
+
+  if (useYarn) {
+    command = 'yarnpkg';
+    args = ['add', '--dev'];
+  } else {
+    command = 'npm';
+    args = ['install', '--save-dev', verbose && '--verbose'].filter(e => e);
+  }
+
```
Comment (Drache93) [inline packages/react-scripts/scripts/init.js]: I don't see any harm in keeping the automatic git init. Anything wrong with keeping it?

File: packages/react-scripts/scripts/init.js (hunk 1 above)
Comment (miraage) [reply packages/react-scripts/scripts/init.js]: Good catch!

File: yarn.lock (hunk 2 above)
Comment (miraage) [reply yarn.lock]: I got the idea. I'll try to come up with smth working

File: packages/react-scripts/scripts/init.js (hunk 3 above)
Comment (miraage) [reply packages/react-scripts/scripts/init.js]: Good catch!

File: packages/react-scripts/scripts/init.js (hunk 1 above)
Comment (Drache93) [reply packages/react-scripts/scripts/init.js]: Sadly :(, other formats are a bit ugly.

We don't need a `.prettierrc`; setting it to `@hiro-ui/prettier-config` works.

**Comment by Drache93** [comment]
Changes look good, should be able to delete the top level `.prettierrc.js` now.

**Pull request merged**